source/apis/routes.py

```diff
@@ -1,13 +1,32 @@
 from pathlib import Path
 
-from fastapi import APIRouter, Response
+import fitz
+from fastapi import APIRouter, Response, Query
 
-from source.apis.utils import generate_pdf, get_available
+from source.services.utils import Aradocgen
 
 router = APIRouter()
 
 text = Path(r"C:\Users\skand\PycharmProjects\DocGeneratorFastApi\source\models\data.txt").read_text(encoding="utf-8")
 
+fonts = \
+    {
+        "Advertising_Bold": r"C:\Users\skand\PycharmProjects\DocGeneratorFastApi\source\models\Arabic Fonts\Advertising_Bold.ttf",
+        "Af_Diwani": r"C:\Users\skand\PycharmProjects\DocGeneratorFastApi\source\models\Arabic Fonts\AF_Diwani.ttf",
```

Comment:
Why do we have fonts here in the route and the service? in utils create a dict called font_mapper!

Also make sure to handle the exception when the font is not used, for example dict["not used font"] will return key error, you need to catch it and raise your own custom error called FontNotUsed, this error is then caugh at the route layer and you raise an HTTP exception.